doc/19_Development_Tools_and_Details/29_Testing/01_Application_Testing.md

```diff
@@ -342,7 +342,7 @@ require_once PIMCORE_PROJECT_ROOT . '/vendor/autoload.php';
 // but is done here for demonstration purpose
 require_once PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support/Util/Autoloader.php';
 
-Autoloader::addNamespace('Pimcore\Tests', PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support');
+Autoloader::addNamespace('Pimcore\Tests', PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests');
```

Comment:
Thank you for the PR!

By looking at
https://github.com/pimcore/data-hub/blob/7652b48a906f466cba4c2aa1d6e44d5cf4d59e7b/tests/_bootstrap.php#L52-L53
for Pimcore 11, it should be like
```suggestion
Autoloader::addNamespace('Pimcore\Tests\Support', PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support');
```